src/components/UniversalNavbar/NavLink.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-// UPDATE anchor tags to NavLink when /term and /login is an internal link in CMS
+import { termHref, accountHref } from './constants'
+
+/**
+ * Checks if provided internal LinkComponent can be used with href
+ * Specifc Link component's should not be using /term or /login
+ * @param {string} href
+ * @return {boolean}
+ */
+const validHRef = (href) => {
+  return ![termHref, accountHref].includes(href)
+}
+
 const NavLink = ({ href, children, LinkComponent, ...props }) => {
-  if (LinkComponent) {
+  if (LinkComponent && validHRef(href)) {
```

Comment:
I don't think we need this and it might conflict with https://github.com/getethos/ethos-design-system/pull/279. 
There is no reason right now that the Gatsby `Link` component we're passing in here wouldn't be able to handle `/term` or `/login`. 

`https://ethoslife.com/term`
`https://ethoslife.com/login`

Both work fine and Mono doesn't currently specify a `LinkComponent` anywhere that I can see in frontend.
```
/ethos/frontend/shared/components/ErrorPageWrapper/ErrorPageWrapper.js:
    1  import React from 'react'
    2  import PropTypes from 'prop-types'
    3: import { Spacer, TitleLarge, UniversalNavbar } from 'ethos-design-system'
    4  import { createRoot } from 'shared/webApp/roots'
    5  
    .
    9    return (
   10      <React.Fragment>
   11:       <UniversalNavbar logoHref={createRoot('cms')} />
   12        <div className={styles.root}>
   13          <div className={styles.centeredBox}>

/ethos/frontend/src/main/accountPages/AccountLayout/AccountLayout.js:
    7    MobileAccountMenu,
    8  } from 'src/main/accountPages/components/AccountMenu/AccountMenu'
    9: import { UniversalNavbar } from 'ethos-design-system'
   10  import { route } from 'shared/webApp/urls'
   11  import { partnerRoute } from 'src/main/partner/urls/partnerRoute'
   ..
   41    return (
   42      <div>
   43:       <UniversalNavbar logoHref={createRoot('cms')} />
   44        <main className={`Container ${styles.container}`}>
   45          <HeaderTitle>My Account</HeaderTitle>

/ethos/frontend/src/main/InsuranceApp/Needs/Landing/index.js:
    9    Spacer,
   10    TitleXLarge,
   11:   UniversalNavbar,
   12  } from 'ethos-design-system'
   13  
   ..
   17    <div className={styles.root}>
   18      <div className={styles.nav}>
   19:       <UniversalNavbar logoHref={createRoot('cms')} />
   20      </div>
   21  

/ethos/frontend/src/main/partner/Preapply/Quote/QuoteTemplate.js:
   11    Media,
   12    TitleMedium,
   13:   UniversalNavbar,
   14    Spacer,
   15  } from 'ethos-design-system'
   ..
   80          )}
   81  
   82:         <UniversalNavbar
   83            hideMobileCta
   84            hideDesktopCta

8 matches across 4 files
```

Reply:
This makes sense. I was reading this doc: https://www.gatsbyjs.org/docs/gatsby-link/#reminder-use-link-only-for-internal-links. 
I always thought the /login and /term was not an internal link

Reply:
They are internal since we added routing on the CMS before user is redirected to App. 

See here:
https://github.com/getethos/cms/blob/master/src/templates/app-template.js
https://github.com/getethos/cms/blob/master/src/content/pages/term.md
https://github.com/getethos/cms/blob/master/src/content/pages/login.md
https://github.com/getethos/cms/blob/master/src/content/pages/app.md